etc/kayobe/environments/ci-aio/stackhpc-ci.yml

```diff
@@ -80,3 +80,10 @@ stackhpc_docker_registry_password: !vault |
   38333133393730633666613965653364316162353337313330346164303631313731646461363461
   3963323635373866630a633533376339363734626664333765313665623662613764363038383735
   38646138376438643533376161376634653439386230353365316239613430363338
+
+# Overcloud host image tags
+stackhpc_centos_8-stream_overcloud_host_image_version: "yoga-20230525T095243"
```

Comment:
hmm, are variables with hyphens valid?

Reply:
It appears to be for now but might result in some unexpected behaviour down the line. I'll have another look at this